broker-daemon/models/market-event.js

```diff
@@ -8,12 +8,13 @@ class MarketEvent {
     this.orderId = orderId
     this.timestamp = timestamp
     this.eventType = eventType
+    this.eventOrder = eventOrder
     this.payload = payload
     this.sep = this.constructor.sep
   }
 
   get key () {
-    return `${this.timestamp}${this.sep}${this.eventId}`
+    return `${this.timestamp}${this.sep}${this.eventId}${this.sep}${this.eventOrder}`
```

Comment:
Maybe we should put this.eventOrder after timestamp so it provides ordering value.